ZeroWasteFriendly/Repository/ZeroWastePlacesRepository.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ZeroWastePlacesRepository.swift
+//  ZeroWasteFriendly
+//
+//  Created by Marek Skrzelowski on 21/02/2020.
+//  Copyright © 2020 vandermesis. All rights reserved.
+//
+
+import Foundation
+import CoreLocation
+
+typealias FetchZeroWastePlacesCompletion = (Result<[ZeroWastePlace], Error>) -> Void
+
+protocol ZeroWastePlacesRepository {
+    func fetchZeroWastePlaces(completion: FetchZeroWastePlacesCompletion?)
+}
+
+final class ZeroWastePlacesRepositoryImpl: ZeroWastePlacesRepository {
+
+    func fetchZeroWastePlaces(completion: FetchZeroWastePlacesCompletion?) {
+        completion?(.success([ZeroWastePlace(name: "Drogeria Hebe",
```

Comment:
How about putting this Mocks to a Mock struuct just like we do for tests, but for now adding in the target for production app and use it here